### PR TITLE
Fixed PlayZone links

### LINKED ISSUE #63: Navbar playzone links modification
Currently in the navbar `Playzone` section, when one clicks, it takes them to the algorithm's main page, which is not the desired result.
```
Current behavior: 
                 The navbar playzone section takes the user to the algorithm's main page.
Desired behavior: 
                 It should take the user to the playground of that particular algorithm.  
                 In the below image, when the user clicks on the Play With Line, it should take the user  
                 to Linear Regression's Playground.
```
Screenshot_36

### PR DESCRIPTION
**Pull request by kunal2812**
Fixes #63

### CODE REVIEW

File: src/components/layout/NavBar.js [hunk 1]
```diff
@@ -37,12 +37,12 @@ export default class NavBar extends Component {
...
-          <Link style={{height: "2rem"}} to="/regression">Play with line</Link>
-          <Link style={{height: "2rem"}} to="/categoricalClassification">Cluster Musrer</Link>
-          <Link style={{height: "2rem"}} to="/objectClassification">Object Classification</Link>
-          <Link style={{height: "2rem"}} to="/imageSegmentation">Image Seg</Link>
-          <Link style={{height: "2rem"}} to="/thug">Thug Life</Link>
-          <Link style={{height: "2rem"}} to="/imageClassification">Image Classification</Link>
+          <Link style={{height: "2rem"}} to="/regression/Playground">Play with line</Link>
+          <Link style={{height: "2rem"}} to="/categoricalClassification/Playground">Cluster Musrer</Link>
+          <Link style={{height: "2rem"}} to="/objectClassification/Playground">Object Classification</Link>
+          <Link style={{height: "2rem"}} to="/imageSegmentation/Playground">Image Seg</Link>
```
Comment (m1-key) [inline src/components/layout/NavBar.js]: Change `Image Seg` to `Image Segmentation`

File: src/components/layout/NavBar.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ export default class NavBar extends Component {
           <Link style={{height: "2rem"}} to="/tut" >Linear Regression</Link>
           <Link style={{height: "2rem"}} to="#">Polynomial Regression</Link>
           <Link style={{height: "2rem"}} to="#">Logistic Regression</Link>
-          <Link style={{height: "2rem"}} to="#">K mean clistering</Link>
+          <Link style={{height: "2rem"}} to="#">K mean clustering</Link>
```
Comment (m1-key) [inline src/components/layout/NavBar.js]: Remove this as we don't have any K Mean Clustering.

**Comment by m1-key** [comment]
@kunal2812 make the changes that have been asked to do in the above review as soon as possible.

**Comment by kunal2812** [comment]
@m1-key I already did sir, what did I miss ?

**Comment by m1-key** [comment]
> @m1-key I already did sir, what did I miss ?

Comments that I have added

**Review by m1-key — CHANGES_REQUESTED** [review]

**Comment by kunal2812** [comment]
Please review.

**Pull request merged**